web/styles/popovers.css

```diff
@@ -772,7 +772,7 @@ ul.popover-group-menu-member-list {
     /* 300px of GIPHY grid + 5px is the extra gutter space
      * + 1px for outline.
      * between gif columns. */
-    width: 306px;
+    width: 506px;
```

Comment:
We might want to set these values as variables in `web/styles/app_variables.css`, since at least the `width` value gets reused in the media query below.